core/modules/field_ui/tests/field_ui.test

```diff
@@ -846,6 +846,67 @@ class FieldUIAlterTestCase extends BackdropWebTestCase {
   }
 }
 
+/**
+ * Tests for correct messaging, and DB cleanup when field creation fails.
+ *
+ * @see based on FieldUIManageFields class.
+ */
+class FieldUIManageFieldsFailTestCase extends BAckdropWebTestCase {
+  function setUp() {
+    parent::setUp();
+
+    $admin_user = $this->backdropCreateUser(array('access content', 'administer content types', 'administer users'));
```

Comment:
Tests are failing because your `$admin_user` also needs the `'administer fields'` permission.

Reply:
I rely on ZenCI for tests, and you know how it has a mind of its own 😓 ...thanks for the recommendation; I've added that permission.